ui/app/src/models/item.ts

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Model name (file name) should be upper case

Reply:
Pending change.

Reply:
done